java/src/main/java/org/rocksdb/OccValidationPolicy.java

```diff
@@ -0,0 +1,40 @@
+//  Copyright (c) Meta Platforms, Inc. and affiliates.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+package org.rocksdb;
+
+public enum OccValidationPolicy {
+
+  // Validate serially at commit stage, AFTER entering the write-group.
```

Comment:
Can you make these comments on the enum values Javadoc rather than comments please?